AxonDeepSeg/segment.py

```diff
@@ -9,7 +9,7 @@
 
 import AxonDeepSeg
 from AxonDeepSeg.apply_model import axon_segmentation
-import os, json
+import os, json, imageio
```

Comment:
has this module been added to the python requirements?

Reply:
Yes.